masjid_util.py

```diff
@@ -0,0 +1,46 @@
+_MASJID_METADATA = {
+  'MCA': {
```

Comment:
i actually disagree here since we are going to have to maintain the enum as well as this list whenever we add more masajids .. i think this makes sense the way it is